docker/data/zkevm/node.toml

```diff
@@ -148,15 +146,15 @@ CleanupLockedProofsInterval = "2m"
 GeneratingProofCleanupThreshold = "10m"
 UpgradeEtrogBatchNumber = 0
 SettlementBackend = "agglayer"
-AggLayerTxTimeout = "5m"
+AggLayerTxTimeout = "10s"
 AggLayerURL = "http://agglayer:4444"
 SequencerPrivateKey = { Path = "/pk/sequencer.keystore", Password = "testonly" }
+BatchProofL1BlockConfirmations = 2
 
 [EthTxManager]
 ForcedGas = 0
 PrivateKeys = [
 	{Path = "/pk/sequencer.keystore", Password = "testonly"},
-	{Path = "/pk/aggregator.keystore", Password = "testonly"}
```

Comment:
Should we revert this maybe?

Reply:
Fix: https://github.com/0xPolygon/agglayer/pull/90/commits/986390a1c514a7a2bdc6730c34a9c9e78f3d9bd1